code/messaging/src/main/java/com/adobe/marketing/mobile/Message.java

```diff
@@ -79,4 +81,13 @@ public interface Message {
      * @param enabled {@code boolean} if true, tracking is done automatically for {@code Message} show, dismiss, and triggered events.
      */
     void setAutoTrack(boolean enabled);
+
+    /**
+     * Gets the {@link Message}'s auto tracking preference.
+     *
+     * @return {@code boolean} containing the auto tracking preference.
+     */
+    default boolean getAutoTrack() {
+        return autoTrack;
```

Comment:
Is true the default value? If so, can you remove the `autoTrack` variable and just return "true"

Reply:
makes sense as true is default.